stripe/src/main/java/com/stripe/android/exception/InvalidRequestException.java

```diff
@@ -7,15 +7,32 @@
 public class InvalidRequestException extends StripeException {
 
     private final String param;
+    private final String code;
+
+    public InvalidRequestException(String message,
+                                   String param,
+                                   String code,
+                                   String requestId,
+                                   Integer statusCode,
+                                   Throwable e) {
+        super(message, requestId, statusCode, e);
+        this.param = param;
+        this.code = code;
+    }
 
     public InvalidRequestException(String message, String param, String requestId, Integer
```

Comment:
this can call `this(message, param, null, requestId, statusCode, e);`

Reply:
Good point